components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-details/public/js/details.js

```diff
@@ -306,17 +306,46 @@ function updateLifecycleHandler(event) {
     };
     promised_update.then(
         (response, event = event_data) => {
-            var message = "Life cycle state updated successfully!";
-            noty({
-                text: message,
-                type: 'success',
-                dismissQueue: true,
-                progressBar: true,
-                timeout: 5000,
-                layout: 'topCenter',
-                theme: 'relax',
-                maxVisible: 10,
-            });
+            console.log(JSON.stringify(response))
+            var lcResponse = response.obj;
+            var jsonPayload = lcResponse.jsonPayload;
+                            
+            if(jsonPayload != null && jsonPayload != "") {
```

Comment:
Same here as well. Better to use if(!jsonPayload) for null checks rather than using if(jsonPayload != null)